entities/component.py

```diff
@@ -112,7 +112,7 @@ def __init__(self, uuid: str, symbol_uuid: SymbolUUID, position: Position,
         self.rotation = rotation
         self.required = required
         self.suffix = suffix
-        self.pins = []  # type: List[PinSignalMap]
+        self.pins: List[PinSignalMap] = []
```

Comment:
When we drop Python 3.8 support we can use `list[...]` directly instead of the `List` wrapper.